docs/technical_reference/flowsheets/seawater_RO_desalination.rst

```diff
@@ -0,0 +1,246 @@
+Seawater RO Desalination
+========================
+
+Introduction
+------------
+
+This flowsheet represents a full-scale seawater reverse osmosis treatment facility.
+The flowsheet includes pretreatment, desalination, post-treatment, and waste handling unit processes available in WaterTAP.
+Unit models used on this flowsheet span the spectrum from simple to detailed.
+
+
+Implementation
+--------------
+
+This flowsheet uses several different modeling features available in WaterTAP, including:
+
+
+WaterTAP costing package
+    * :doc:`/technical_reference/costing/watertap_costing`
+Unit model costing packages
+    * :doc:`/technical_reference/costing/detailed_unit_model_costing`
+Zero order unit and costing models
+    * :doc:`/technical_reference/unit_models/zero_order_unit_models/index`
+Zero order costing package
+    * :doc:`/technical_reference/costing/zero_order_costing`
+Zero order property package
+    * :doc:`/technical_reference/core/water_props`
+Seawater property package
+    * :doc:`/technical_reference/property_models/seawater`
+Reverse osmosis model
+    * :doc:`/technical_reference/unit_models/reverse_osmosis_0D`
+Pressure exchanger model
+    * :doc:`/technical_reference/unit_models/pressure_exchanger`
+IDAES Translator blocks
+    * :doc:`idaes:reference_guides/model_libraries/generic/unit_models/translator`
+IDAES Product blocks
+    * :doc:`idaes:reference_guides/model_libraries/generic/unit_models/product`
+IDAES Separator blocks
+    * :doc:`idaes:reference_guides/model_libraries/generic/unit_models/separator`
+IDAES Mixer blocks
+    * :doc:`idaes:reference_guides/model_libraries/generic/unit_models/mixer`
```

Comment:
I feel like this section is a bit intimidating to look at lol. Maybe just group some of these together like:

Costing Packages
- WaterTAP Costing
- ZO Costing
- Unit Model Costing

Unit Models
- Mixer...

Reply:
Yeah agreed and good idea